core/serialization/serialization.js

```diff
@@ -97,6 +97,20 @@ var Serialization = Montage.specialize( /** @lends Serialization.prototype # */
         }
     },
 
+    removeObject: {
+        value: function (label) {
+            var serializationObject = this.getSerializationObject();
+
+            if (serializationObject && label in serializationObject) {
+                delete serializationObject[label];
+
+                this.initWithObject(serializationObject);
```

Comment:
Is it needed to re-do a complete .initWithObject() It's unusual to do that. Performance impact? Side effects on current state?

Reply:
Agreed maybe not the safest way of doing it .